app/containers/Products/ProductRowItemContainer.tsx

```diff
@@ -147,6 +148,7 @@ export const ProductRowItemComponent: React.FunctionComponent<Props> = ({
       centered
       size="xl"
       show={benchmarkModalShow}
+      aria-labelledby="product-benchmark-header"
```

Comment:
We probably want to put this identifier (and the one just above) in a static export from `InnerProductBenchmarkModal`, so we don't have to worry about this changing in one or the other place